pkg/infra/infra.go

```diff
@@ -253,7 +253,7 @@ var _ = Describe("Managed cluster should", framework.LabelMachines, func() {
 				klog.Infof("Did not find all expected taints on the node. Missing: %v", expectedTaints.Difference(observedTaints))
 
 				return false
-			}, framework.WaitMedium, 5*time.Second).Should(BeTrue())
+			}, framework.WaitMedium, 5*time.Second).Should(BeTrue(), "Should find all the expected taints on the Node")
```

Comment:
As a future todo, we should rewrite this with Komega so that we can use gomega assertions to compare the desired taints, as that would allow better output than a bool here

Reply:
+1